hyperwallet/hyperwallet-android-ui-sdk
# HW-58541 valueChanged on widget now only validates single item traver…

https://jira.site1.hyperwallet.local/browse/HW-58541
- changed interface of valueChanged to add AbstractWidget as argument, so that we can validate a single item

Tested
- check to see Insights called trackError on the last focused item only
- check submit form validates the entire form

https://jira.site1.hyperwallet.local/browse/HW-54774
 - scroll to first error

https://jira.site1.hyperwallet.local/browse/HW-58840
 - country pop up fix for API <= 27
